WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -638,6 +638,7 @@ class AbstractPostListViewController: UIViewController,
         options.authorID = author
         options.number = numberOfLoadedElement
         options.purgesLocalSync = true
+        options.meta = "autosave"
```

Comment:
I just realized that option 2 may not be ideal since `PostServiceSyncOptions` just conforms to `PostServiceRemoteOptions`. 🙁 Anyway, let me know what you think.

Reply:
[`meta` takes a comma separated list used for `sites`](https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/posts/), so I think it makes sense for it to be in `PostServiceSyncOptions` to allow it to be customized.

I agree that expecting clients to set `autosave` is problematic - if we set a default value in `PostServiceSyncOptions` however, clients must remember to include `autosave` when for example, fetching likes:

`meta = "likes,autosave"`

We can _always_ include autosave in [`- dictionaryWithRemoteOptions:`](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/30b720caf3c96792f6a004752f9bcf0082f839a8/WordPressKit/PostServiceRemoteREST.m#L308), no matter what `meta` is set to. This presumes we will always want `autosave` included.

I'll go with a default value in `PostServiceSyncOptions` as customizing the `meta` property with anything but autosave is not the majority use case at the moment. Let me know if you have any concerns.